### ISSUE TITLE
Use Django Crispy Forms to render forms

### ISSUE THREAD

**Issue by Laur04** [tjcsl/ion#1075]
## Feature description
Django Crispy Forms template filters render forms with `div`s pretty styles. It's worth looking at.

## Rationale for adding feature
It could possibly make our table based forms `annoucements` and `itemreg` for example look better.

## Suggested implementation(s) (if applicable)
- See [this](https://django-crispy-forms.readthedocs.io/en/latest/filters.html)

**Comment by Gimberk** [comment]
Hey, could I be assigned to this?

**Comment by Pwngo** [comment]
May I be assigned this issue?

**Comment by aarushtools** [comment]
yes

**Comment by Pwngo** [comment]
Adding crispy forms alone won't improve the look of the forms, since Django crispy forms require a template built on a CSS framework (Tailwind, Bootstrap, Bulma, etc.), which we don't have. Additionally, a framework will break ION's custom CSS styles, and adding a custom crispy forms template only changes the HTML surrounding the form, so nothing changes visually. I think the best course of action is to manually rewrite the CSS to use div instead of table. I can do that for announcements and itemreg, but I'm unsure which course of action is best.

**Comment by aarushtools** [comment]
You're right, and on second thought I don't think this issue is necessary to fix - the forms look fine.
Try looking into another good first issue, maybe #1815
